ctk/idp/src/main/kotlin/org/codice/compliance/web/sso/RedirectSSOTest.kt

```diff
@@ -130,14 +135,49 @@ class RedirectSSOTest : StringSpec() {
             }
         }
 
+        "Redirect AuthnRequest Using DSA1 Signature Algorithm" {
```

Comment:
I will update the test with description and spec reference.

Reply:
For the negative tests I did `Doc section: test name` like `Core 3.1.4: POST incorrect x tests` and same for the log message below